examples/eth.rs

```diff
@@ -0,0 +1,65 @@
+use esp_idf_sys::{self as _}; // If using the `binstart` feature of `esp-idf-sys`, always keep this module imported
+
+#[cfg(esp32)]
+use esp_idf_svc::{
+    eth::{BlockingEth, EspEth, EthDriver},
+    eventloop::EspSystemEventLoop,
+    hal::{gpio, prelude::Peripherals},
+    log::EspLogger,
+};
+#[cfg(esp32)]
+use log::info;
+
+#[cfg(esp32)]
+fn main() -> anyhow::Result<()> {
+    esp_idf_sys::link_patches();
+    EspLogger::initialize_default();
+
+    let peripherals = Peripherals::take().unwrap();
+    let pins = peripherals.pins;
+    let sys_loop = EspSystemEventLoop::take()?;
+
+    // Make sure to configure ethernet in sdkconfig and adjust the parameters below for your hardware
+    let eth_driver = EthDriver::new(
+        peripherals.mac,
+        pins.gpio25,
+        pins.gpio26,
+        pins.gpio27,
+        pins.gpio23,
+        pins.gpio22,
+        pins.gpio21,
+        pins.gpio19,
+        pins.gpio18,
+        esp_idf_svc::eth::RmiiClockConfig::<gpio::Gpio0, gpio::Gpio16, gpio::Gpio17>::OutputInvertedGpio17(
+            pins.gpio17,
+        ),
+        Some(pins.gpio5),
+        esp_idf_svc::eth::RmiiEthChipset::LAN87XX,
```

Comment:
Out of pure curiosity - is this working? Another user on the matrix room had issues with precisely this chipset, you might want to chime in there for help.

Reply:
Yes, this is working on Olimex ESP32-Gateway, but I didn't do anything special other than using these specific pins.